cmd/kubemark/hollow-node.go

```diff
@@ -92,7 +92,7 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.BootstrapKubeconfigPath, "bootstrap-kubeconfig", "", "Path to bootstrap kubeconfig file.")
 	fs.StringVar(&c.CertDirectory, "cert-dir", "/etc/srv/", "Path to cert directory for bootstraping.")
 	fs.IntVar(&c.KubeletPort, "kubelet-port", ports.KubeletPort, "Port on which HollowKubelet should be listening.")
-	fs.IntVar(&c.KubeletReadOnlyPort, "kubelet-read-only-port", ports.KubeletReadOnlyPort, "Read-only port on which Kubelet is listening.")
+	fs.IntVar(&c.KubeletReadOnlyPort, "kubelet-read-only-port", 0, "Read-only port on which Kubelet is listening.")
```

Comment:
```suggestion
	fs.IntVar(&c.KubeletReadOnlyPort, "kubelet-read-only-port", 0, "Read-only port on which Kubelet is listening. The value `0` indicates that the port is disabled.")
```

this may be a valuable addition as some systems uses `0` to pick a random

Reply:
Updated.